homeassistant/components/homekit/__init__.py

```diff
@@ -61,6 +68,11 @@
     validate_media_player_features,
 )
 
+# TODO: switch the below once done testing outside of tree
+# from homeassistant.components.binary_sensor import DEVICE_CLASS_BATTERY_CHARGING
```

Comment:
To do.

Reply:
I'm going to do the binary sensor changes in a separate merge since it requires an architecture doc